new_platforms/opteesim/Trusted/cyres_sim.c

```diff
@@ -16,11 +16,11 @@
 #define DICE_UDS_LENGTH 0x20
 
 /* Estimate the buffer size needed, in bytes, for a TCPS ID. In our case
-   we will not have more then two PUBKEYs and a FW measurement. */
-#define CLAIM_BUFFER_LENGTH (TCPS_ID_PUBKEY_LENGTH * 2) + TCPS_ID_FWID_LENGTH
+   we will not have more then two PUBKEYs, a FW measurement and some metadata. */
+#define CLAIM_BUFFER_LENGTH ((TCPS_ID_PUBKEY_LENGTH * 2) + TCPS_ID_FWID_LENGTH + 23)
```

Comment:
Can you define magic metadata length "23"?

Reply:
Done.